jekyll/_cci2/language-clojure.md

```diff
@@ -102,7 +102,7 @@ Directly beneath `working_directory`, we can specify container images under a `d
 version: 2
 ...
     docker:
-      - image: circleci/clojure:lein-2.9.1
+      - image: cimg/clojure:lein-2.9.1
```

Comment:
The tag `lein-2.9.1` doesn't exist. We can use `1.10`.

Reply:
Fixed